crates/api_models/src/payments.rs

```diff
@@ -454,6 +454,8 @@ pub enum PayLaterData {
         #[schema(value_type = String)]
         billing_name: Secret<String>,
     },
+    PayBright {},
```

Comment:
These payment methods don't have any mandatory parameters? what about billing address?

Reply:
@Narayanbhat166 , yes these payment method doesn't required mandatory fields. I cross verified with checkout, stripe and Aden docs (whatever the payment methods applicable)